tests/test_lab_config.py

```diff
@@ -16,3 +20,93 @@ def test_ilab_config_show(cli_runner: CliRunner) -> None:
     assert parsed
 
     assert configuration.Config(**parsed)
+
+
+def test_ilab_config_init_with_env_var_config(
+    cli_runner: CliRunner, tmp_path: pathlib.Path
+) -> None:
+    # create a dummy config file
+    cfg = configuration.get_default_config()
+    assert cfg.general.log_level == "INFO"
+    cfg.general.log_level = "DEBUG"
+    cfg_file = tmp_path / "config.yaml"
+    with cfg_file.open("w") as f:
+        yaml.dump(cfg.model_dump(), f)
+
+    result = cli_runner.invoke(
+        lab.ilab, ["config", "init"], env={"ILAB_GLOBAL_CONFIG": cfg_file.as_posix()}
+    )
+    assert result.exit_code == 0, result.stderr
+
+    # the following feels ugly, if someone has a better way to extract the path from the output
```

Comment:
I probably miss something, but why do you even need to extract the path from the output? Can't you just read the file from the CONFIG_DIR?

```
extracted_path = pathlib.Path(configuration.DEFAULTS.CONFIG_FILE)
```

This whole block seems unnecessary.